.github/workflows/tag.yml

```diff
@@ -3,15 +3,23 @@ on:
   workflow_dispatch:
   workflow_call:
     inputs:
+      dry-run:
+        description: "Determine the next version without tagging the branch. The workflow can use the outputs new_tag and tag in subsequent steps. Possible values are true and false (default)"
+        default: false
+        required: false
+        type: string
       print-tag:
-        description: "Echo generated tag to console"
-        default: "true"
+        description: "Echo tag to console"
+        default: true
         required: false
         type: string
     outputs:
       tag:
-        description: "Generated tag"
+        description: "The value of the latest tag after running this action"
         value: ${{ jobs.tag-job.outputs.tag }}
+      new-tag:
```

Comment:
I don't understand the difference between `new-tag` and `tag`. If `tag` is the latest tag, then wouldn't that be equivalent to `new-tag`? As it currently stands, I wouldn't know which value to use if I needed to reference a tag in a GHA

Reply:
This is how it goes. The `new-tag` is always the tag resulting from a bump to the original tag. However, the `tag`is by definition the value of the latest tag after running the action, which might not change if dry run is used, and remains same as the original tag.

Reply:
I have incorporated more comments about `tag` vs. `new-tag`.